### PR TITLE
Sign redux

### PR DESCRIPTION
**Pull request by Surfsol**

### CODE REVIEW

**Comment by Surfsol** [comment]
I added a new component, SignIn.js.  Uses redux and material-ui sign in form.  On App.js this file will replace Landing.

**Comment by Surfsol** [comment]
Also, I see the user type was not returned after login.  Perhaps I have the wrong endpoint. Please check.

File: src/components/Map/Map.component.jsx [hunk 1]
```diff
@@ -20,9 +21,10 @@ export default function Map(props) {
 
   return (
     <div>
+      
       <ReactMapGl
         mapboxApiAccessToken={
-          'pk.eyJ1IjoiaHRyYW4yIiwiYSI6ImNrMmdmeWM2dDB1amkzY3AwNWgwNHRteXUifQ.jG0OQ6bMhr-sZYMkdj3H6w'
+          process.env.REACT_APP_MAPBOX_TOKEN
```
Comment (luiscmartinez) [inline src/components/Map/Map.component.jsx]: 🕶

**Comment by luiscmartinez** [comment]
* signin reducer
* signin view page => form constrained.

**Pull request merged**